### PR TITLE
Added explaination for merge-sort

### PR DESCRIPTION
**Pull request by naveenbanda**

### CODE REVIEW

File: src/pages/algorithms/sorting-algorithms/merge-sort/index.md [hunk 1]
```diff
@@ -3,13 +3,30 @@ title: Merge Sort
 ---
 ## Merge Sort
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/algorithms/sorting-algorithms/merge-sort/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Merge Sort is a <a href='https://github.com/naveenbanda/guides/tree/master/src/pages/algorithms/divide-and-conquer-algorithms' target='_blank' rel='nofollow'>Divide and Conquer</a> algorithm. It divides input array in two halves, calls itself for the two halves and then merges the two sorted halves.
```
Comment (erictleung) [inline src/pages/algorithms/sorting-algorithms/merge-sort/index.md]: Please change the link to `https://guide.freecodecamp.org/algorithms/divide-and-conquer-algorithms`

File: src/pages/algorithms/sorting-algorithms/merge-sort/index.md [hunk 2]
```diff
@@ -3,13 +3,30 @@ title: Merge Sort
...
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+```Algorithm
+MergeSort(arr[], left,  right):
+If right > l:
+     1. Find the middle point to divide the array into two halves: 
+             mid = (left+r)/2
```
Comment (erictleung) [inline src/pages/algorithms/sorting-algorithms/merge-sort/index.md]: I think the `r` should be `right`?

File: src/pages/algorithms/sorting-algorithms/merge-sort/index.md [hunk 3]
```diff
@@ -3,13 +3,30 @@ title: Merge Sort
...
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+### Properties:
+* Space Complexity: O(n)
+* Time Complexity: O(n*log(n))
+* Sorting In Place: No in a typical implementation
+* Stable: Yes
+
+### Relavant videos on freeCodeCamp YouTube channel
+* <a href='http://www.mycodeschool.com/videos/merge-sort-algorithm' target='_blank' rel='nofollow'>Merge Sort algorithm - MyCodeSchool</a>
```
Comment (erictleung) [inline src/pages/algorithms/sorting-algorithms/merge-sort/index.md]: This link doesn't allow you to play the video straight from the website. Please change the link to the YouTube video itself 

```
https://youtu.be/TzeBrDU-JaY
```

**Comment by texas2010** [comment]
@erictleung, ping you. it seems he did change it as you requested.

**Comment by erictleung** [comment]
@texas2010 thanks for the ping :smile:

**Comment by naveenbanda** [comment]
@erictleung You are Welcome!! Will try to do the same in future.

**Pull request merged**